### PR TITLE
Pipewire connection fix

### PR DESCRIPTION
**Pull request by x37v**

### CODE REVIEW

**Comment by x37v** [comment]
@sadguitarius I've made a pull request as that is a nice way to review this stuff, first glance, this looks good! I'll have to run it and see.

File: src/components/editor/port.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const EditorPort: FunctionComponent<PortProps> = memo(function WrappedPort({
 			id={ port.id }
 			position={ handlePositionByPortDirection[port.direction] }
 			data-c74-type={ port.type }
-			data-c74-name={ alias || port.id }
+			data-c74-name={ (alias || port.id).replace(/\((capture|playback)_[0-9+]\)/, "") }
```
Comment (x37v) [inline src/components/editor/port.tsx:32]: Hmm.. so, if a port has no alias and is just called `"capture_1"` then it just becomes `""` ?

**Comment by sadguitarius** [comment]
Cool, let me know if you need any details from my pipewire config to get this working right. At least this part of the jack config might be helpful:

```
jack.properties = {
     jack.show-midi     = true
     jack.short-name    = true
}
```

File: src/components/editor/port.tsx (hunk 1 above)
Comment (sadguitarius) [reply src/components/editor/port.tsx:32]: ah, I think I tried only applying the regex to alias and was getting an error, I'll look at that more closely.

Comment (sadguitarius) [reply src/components/editor/port.tsx:32]: Oh whoops I had it flipped around. Does this work? 
`data-c74-name={ (alias || port.id.replace(/\((capture|playback)_[0-9+]\)/, "")) }`

Comment (x37v) [reply src/components/editor/port.tsx]: I suspect we'll wanna remove the `[0-9]+` from the regex but i'll have to test it out

Comment (sadguitarius) [reply src/components/editor/port.tsx]: Just so you know, the intention was to only remove the labels added by pipewire-jack containing capture or playback followed by an underscore and a number, enclosed in parentheses. Just trying to prettify the names and remove redundant information. Hopefully I got the regex right and hopefully this is not a pattern that would be commonly used for any other reason.

**Comment by x37v** [comment]
these changes look good! thanks @sadguitarius

**Comment by sadguitarius** [comment]
Awesome, thank you!

**Pull request merged**